homeworks/dmytro.shevchenko_dmytro-shevchenko1/GamesProject/style.css

```diff
@@ -0,0 +1,26 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+main {
+  width: 1175px;
+}
+
+.intro {
+  color: #221F1F;
+  width: 575px;
+  font-size: 22px;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+  line-height: 32px;
+}
+
+h1 {
+  color: #221F1F;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 32px;
+  margin-block-start: 0;
+}
+
+.MA-link {
```

Comment:
better use lowecase for css classes `.ma-link`. 
Or just `.link`, proly all your links will look like this.